app/controllers/simulator_controller.rb

```diff
@@ -1,5 +1,5 @@
 class SimulatorController < ApplicationController
-
+  include SimulatorHelper
```

Comment:
Is it possible to implement this in a similar fashion as the `Utils` has been used in `collaborations_controller` and `group_members_controller`?

Reply:
Yes it is possible but i prefer to do it this way .

Reply:
Okay, I felt it would have been better for the overall uniformity in the way helper functions were defined and used :slightly_smiling_face:

Reply:
We will need a major refactor of all these things probably during the GSoC period. We will need to have guidelines on these things so that there is no confusion.